manala.composer/tests/0110_version.goss.yml

```diff
@@ -0,0 +1,24 @@
+---
+
```

Comment:
You can get rid of packages & unzip tests, as we only check version in this test